cfme/ansible/repositories.py

```diff
@@ -75,6 +75,16 @@ def is_displayed(self):
         )
 
 
+class RepositoryPlaybookView(RepositoryDetailsView):
```

Comment:
does this view show one playbook or many ?

Reply:
It shows All Playbooks, when navigated.

Reply:
So, I guess it should be called RepositoryPlaybooksView, right ?

Reply:
@izapolsk Updated it.